doc/man7/EVP_SIGNATURE-ED25519.pod

```diff
@@ -38,7 +38,7 @@ instance, a nonempty context-string is not permitted.
 
 =head2 ED25519 and ED448 Signature Parameters
 
-Two parameters can be set during signing or verification: the EdDSA
+Three parameters can be set during signing or verification: the EdDSA
```

Comment:
this is not right still.. I think listing the params and then listing them as items is not required.

Reply:
Better?